binzx/otomi

```diff
@@ -207,6 +207,7 @@ vars=(
   AZURE_CLIENT_SECRET
   AZURE_TENANT_ID
   CI
+  CUSTOM_NETWORK
```

Comment:
Please document this env, e.g.:
# CUSTOM_NETWORK - set if otomi container needs to connect services from isolated network, e.g.: minikube